variables.tf

```diff
@@ -3,6 +3,18 @@ variable "name" {
   type        = string
 }
 
+variable "app_version" {
+  description = "Version of transformer kinesis"
+  type        = string
+  default     = "5.8.0"
```

Comment:
To confirm that 5.8.0 has no major config changes needed for this module to still work without the config override?

Reply:
yes, it should work without config override, thanks for the heads up!

I'll prepare them now

Reply:
done!